kafka/consumer_test.go

```diff
@@ -65,6 +64,20 @@ func TestConsumerAPIs(t *testing.T) {
 		t.Errorf("Unsubscribe failed: %s", err)
 	}
 
+	topic := "gotest"
+	stored, err := c.OffsetsStore([]TopicPartition{{Topic: &topic, Partition: 0, Offset: 1}})
+	if err != nil && err.(Error).Code() != ErrUnknownPartition {
+		t.Errorf("OffsetsStore() failed: %s", err)
+		toppar := stored[0]
+		if toppar.Error.(Error).Code() == ErrUnknownPartition {
+			t.Errorf("OffsetsStore() TopicPartition error: %s", toppar.Error)
+		}
+	}
+	stored, err = c.OffsetsStore(nil)
```

Comment:
Let's call it an empty slice. I think that the API should allow passing an empty slice, whatever that means in Go. If someone chooses to pass an empty slice a proper error is returned. We don't explicitly handle it (as in `consumer.go` code) but we explicitly test that an empty `[]TopicPartition` is properly handled.